app/views/account/scaffolding/completely_concrete/tangible_things/targets_many_actions/show.html.erb

```diff
@@ -20,7 +20,7 @@
             <%= render 'shared/attributes/text', attribute: :target_count %>
             <%= render 'shared/attributes/text', attribute: :performed_count %>
             <%= render 'shared/attributes/belongs_to', attribute: :created_by %>
-            <%= render 'shared/attributes/belongs_to', attribute: :approved_by %>
+            <%= render 'shared/attributes/belongs_to', attribute: :approved_by_id %>
```

Comment:
@jagthedrummer Good question, I think I've scratched the surface of the underlying issue as to why this is actually happening. I assumed all of the logic was correctly in place because the `targets-many` test in the Starter Repository is passing. However, upon more investigation, it looks like a few things are at play here.

1. The targets-many strong params is missing the `approved_by` field
2. The error in the show view is raised because [include Actions::RequiresApproval](https://github.com/bullet-train-pro/bullet_train-action_models/blob/ff7f91e988085778430cd548ca25f6beab8a4925/app/models/scaffolding/completely_concrete/tangible_things/targets_many_action.rb#L13) is commented-out by default, but we don't have a safeguard against this when accessing the view.
3. There are different places in the code base where there are discrepancies that don't match concerning `created_by` and `approved_by` and their use of the `_id` suffix. I'll try to report on that if I push a proper fix.

I don't have enough time to fix everything today, but I think this is relatively high-priority because users cannot currently access the show view without a fix in place. I'll try to get back to this tomorrow.

# Prying into the console

Here's an example of what the `created_by` and `approved_by` attributes along with their _id suffix versions look like when not including the `Actions::RequiresApproval` module. 
```
> rails c
DEPRECATED: please use `enable_cable_ready_updates` instead. The `enable_updates` option will be removed from a future version of CableReady 5
Loading development environment (Rails 7.0.8)
irb(main):001> Projects::ArchiveAction.first.created_by_id
  Projects::ArchiveAction Load (0.7ms)  SELECT "projects_archive_actions".* FROM "projects_archive_actions" ORDER BY "projects_archive_actions"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> 1
irb(main):002> Projects::ArchiveAction.first.created_by
  Projects::ArchiveAction Load (1.2ms)  SELECT "projects_archive_actions".* FROM "projects_archive_actions" ORDER BY "projects_archive_actions"."id" ASC LIMIT $1  [["LIMIT", 1]]
  Membership Load (0.5ms)  SELECT "memberships".* FROM "memberships" WHERE "memberships"."id" = $1 LIMIT $2  [["id", 1], ["LIMIT", 1]]
=> 
#<Membership:0x00007faf60c0b888
 id: 1,
 user_id: 1,
 team_id: 1,
 created_at: Thu, 28 Sep 2023 06:45:07.721921000 UTC +00:00,
 updated_at: Thu, 28 Sep 2023 06:45:07.796218000 UTC +00:00,
 invitation_id: nil,
 user_first_name: nil,
 user_last_name: nil,
 user_profile_photo_id: nil,
 user_email: "g-zayas@hotmail.com",
 added_by_id: nil,
 platform_agent_of_id: nil,
 role_ids: ["admin"],
 platform_agent: false>
irb(main):003> Projects::ArchiveAction.first.approved_by_id
  Projects::ArchiveAction Load (1.3ms)  SELECT "projects_archive_actions".* FROM "projects_archive_actions" ORDER BY "projects_archive_actions"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> nil
irb(main):004> Projects::ArchiveAction.first.approved_by
  Projects::ArchiveAction Load (1.0ms)  SELECT "projects_archive_actions".* FROM "projects_archive_actions" ORDER BY "projects_archive_actions"."id" ASC LIMIT $1  [["LIMIT", 1]]
/home/gazayas/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/activemodel-7.0.8/lib/active_model/attribute_methods.rb:450:in `method_missing': undefined method `approved_by' for #<Projects::ArchiveAction id: 1, team_id: 1, target_all: true, target_ids: ["1", "2"], failed_ids: [], last_completed_id: 2, started_at: "2023-09-28 10:50:48.083984000 +0000", completed_at: "2023-09-28 10:50:48.133765000 +0000", target_count: 2, performed_count: 2, scheduled_for: nil, sidekiq_jid: "1f1820ac73e7666312371976", created_by_id: 1, approved_by_id: nil, created_at: "2023-09-28 10:50:47.765664000 +0000", updated_at: "2023-09-28 10:50:48.137977000 +0000"> (NoMethodError)
Did you mean?  approved_by_id
```